src/main/scala/scalaz/netty/Server.scala

```diff
@@ -126,10 +126,9 @@ private[netty] object Server {
 
     //val server = new Server(bossGroup, config.limit)
     val bootstrap = new ServerBootstrap
-    
+
     val serverQueue = async.boundedQueue[(InetSocketAddress, Process[Task, Exchange[ByteVector, ByteVector]])](config.limit)
-    val handlerQueue = async.boundedQueue[ByteVector](config.limit)
```

Comment:
We need a single `handlerQueue` per `ServerHandler` (i.e. per connection) rather than one per `Server`. This way we have control over, for example, closing single connections without closing *all* the connections :) Is that the bug you were fixing?